builtin/commands/dposV2_commands.go

```diff
@@ -111,12 +111,19 @@ func DelegateCmdV2() *cobra.Command {
 				return err
 			}
 
-			return cli.CallContract(DPOSV2ContractName, "Delegate", &dposv2.DelegateRequestV2{
-				ValidatorAddress: addr.MarshalPB(),
-				Amount: &types.BigUInt{
-					Value: *amount,
-				},
-			}, nil)
+            var req dposv2.DelegateRequestV2
+            req.Amount = &types.BigUInt{Value: *amount}
+	        req.ValidatorAddress = addr.MarshalPB()
+
+            if len(args) == 3 {
+			    lockTime, err := strconv.ParseInt(args[2], 10, 64)
+			    if err != nil {
+                    return err
+			    }
+                req.LockTime = uint64(lockTime)
+            }
+
```

Comment:
something strange is happening with the spacing here, check that you're using tabs and not spaces? or that the return below is properly indented.